eng/common/pipelines/templates/steps/create-pull-request.yml

```diff
@@ -86,4 +86,4 @@ steps:
       -UserReviewers "${{ parameters.GHReviewersVariable }}"
       -TeamReviewers "${{ parameters.GHTeamReviewersVariable }}"
       -Assignees "${{ parameters.GHAssignessVariable }}"
-      -CloseAfterOpenForTesting $${{ parameters.CloseAfterOpenForTesting }}
+      -CloseAfterOpenForTesting $${{ coalesce(parameters.CloseAfterOpenForTesting, 'false') }}
```

Comment:
Do we need to coalesce given that we default the value to false?

Reply:
Yes, unless we go set it for all the ci.yml files. Other wise it will resolve to `' '` at times.

Reply:
I'm fine with this change but I am curious what cases would hit this with empty? A pointer would be good.

Reply:
So for regular non-test pipelines. Where no parameter is passed to TestPipeline the value here will resolve to `' '`

Reply:
Why is that? I see on line 22 https://github.com/Azure/azure-sdk-tools/pull/1131/files#diff-2da9c3d63f1a3177d12f9e47cb7a8d973239379a5e86970b335825f663e6c4f1L22 we pass false as a default value so it should be false if nothing is passed.

Reply:
Yes, but if you pass a value to this the default will be overridden. So if when ${{ parameter.TestPipeline }} is passed it will override this default value. Right? And in some case that value may resolve to an empty string "".